ingestion/src/metadata/ingestion/source/dashboard/tableau.py

```diff
@@ -55,6 +55,49 @@
 TABLEAU_TAG_CATEGORY = "TableauTags"
 
 
+class TableauBaseModel(BaseModel):
+    """
+    Tableau basic configurations
+    """
+
+    class Config:
+        extra = Extra.allow
+
+    id: str
+    name: str
+
+
+class TableauOwner(TableauBaseModel):
+    """
+    Tableau Owner Details
+    """
+
+    email: str
+
+
+class TableauChart(TableauBaseModel):
+    """
+    Chart (View) representation from API
+    """
+
+    workbook_id: str
+    sheet_type: str
+    view_url_name: str
+    tags: List[str]
+
+
+class TableauDashboard(TableauBaseModel):
```

Comment:
adding closed definitions to help understand the data flow

Reply:
these are the elements we are currently using. Still configured `Extra.allow` to not break if the response has more data. Whenever we want to push any other feature (e.g., usage), it will help us identify from where we are picking up the data